hardware/Netatmo.cpp

```diff
@@ -1358,7 +1358,7 @@ bool CNetatmo::ParseHomeStatus(const std::string &sResult)
 				{
 					std::string aName = "Status";
 					bool bIsActive = (module["boiler_status"].asString() == "true");
-					SendSwitch(moduleID & 0x00FFFFFF | 0x10000000, 1, 255, (bIsActive == true), 0, aName);
+					SendSwitch((moduleID & 0x00FFFFFF) | 0x10000000, 1, 255, (bIsActive), 0, aName);
```

Comment:
I think we can also remove the ( ) characters here?

Reply:
Yeah I'll do that when I can.

Reply:
Done.